crates/frontend/src/pages/context_override.rs

```diff
@@ -1,77 +1,206 @@
 use crate::api::fetch_config;
 use crate::api::{delete_context, fetch_default_config, fetch_dimensions};
+use crate::components::alert::AlertType;
 use crate::components::button::Button;
-use crate::components::condition_pills::types::Condition;
-use crate::components::condition_pills::ConditionPills;
+use crate::components::context_card::ContextCard;
 use crate::components::context_form::utils::{create_context, update_context};
 use crate::components::context_form::ContextForm;
 use crate::components::drawer::{close_drawer, open_drawer, Drawer, DrawerBtn};
 use crate::components::override_form::OverrideForm;
 use crate::components::skeleton::{Skeleton, SkeletonVariant};
-use crate::components::table::{types::Column, Table};
-use crate::types::{Config, DefaultConfig, Dimension};
+use crate::providers::alert_provider::enqueue_alert;
+use crate::types::{Config, Context, DefaultConfig, Dimension};
 use crate::utils::extract_conditions;
 use futures::join;
 use leptos::*;
 use serde::{Deserialize, Serialize};
-use serde_json::{Map, Value};
+use serde_json::{json, Map, Value};
 
 #[derive(Clone, Debug, Default)]
-pub struct TableData {
+pub struct Data {
     pub context: Vec<(String, String, String)>,
-    pub overrides: Map<String, Value>,
+    pub overrides: Vec<(String, Value)>,
 }
 
-#[derive(Serialize, Deserialize, Clone, Debug)]
-struct CombinedResourceOverride {
-    config: Option<Config>,
+#[derive(Serialize, Deserialize, Clone, Debug, Default)]
+struct PageResource {
+    config: Config,
     dimensions: Vec<Dimension>,
     default_config: Vec<DefaultConfig>,
 }
 
+#[derive(Debug, Clone)]
+enum FormMode {
+    Edit,
+    Create,
+}
+
+#[component]
+fn form(
+    context: Vec<(String, String, String)>,
+    overrides: Vec<(String, Value)>,
+    dimensions: Vec<Dimension>,
+    edit: bool,
+    default_config: Vec<DefaultConfig>,
+    handle_submit: Callback<(), ()>,
+) -> impl IntoView {
+    let tenant_rs = use_context::<ReadSignal<String>>().unwrap();
+    let (context, set_context) = create_signal(context);
+    let (overrides, set_overrides) = create_signal(overrides);
+    let dimensions = StoredValue::new(dimensions);
+
+    let on_submit = move |_| {
+        spawn_local(async move {
+            let f_context = context.get();
```

Comment:
can you check here , that context does not contains empty operator or operand
you can also modify the context_form and add validation there before submit

Reply:
@pratikmishra356 , this is beyond the scope of this PR, as already discussed we have an open issue for this.